Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
This doesn't have anything to do with "public API". By "downstream" I mean code that is consuming the formula API, internal code included. There is no reason that consuming code needs to know these implementation details.

Please undo this change.